platform/platform_tests/downloader_tests/downloader_test.cpp

```diff
@@ -211,6 +211,8 @@ UNIT_TEST(DownloaderSimpleGet)
   }
 }
 
+// TODO: This test sometimes fails on CI. Reasons are unknown.
```

Comment:
Why does it fail? Why it didn't fail before?

Reply:
As mentioned in the comment, "Reasons are unknown".